powershell/extractor/Semmle.Extraction.PowerShell.Standalone/Options.cs

```diff
@@ -70,12 +79,48 @@ public override void InvalidArgument(string argument)
         /// <summary>
         /// Files/patterns to exclude.
         /// </summary>
-        public IList<string> Excludes { get; } = new List<string>() { "node_modules", "bower_components" };
+        public IList<string> Excludes { get; } =
+            new List<string>() { "node_modules", "bower_components" };
+
+        /// <summary>
+        /// Returns a FileInfo object for each file in the given path (recursively).
+        /// </summary>
+        private static FileInfo[] GetFiles(string path)
+        {
+            var di = new DirectoryInfo(path);
+            return di.Exists
+                ? di.GetFiles("*.*", SearchOption.AllDirectories)
+                : new FileInfo[] { new FileInfo(path) };
+        }
+
+        /// <summary>
+        /// Returns a list of files to extract. By default, this is the list of all files in
+        /// the current directory. However, if the LGTM_INDEX_INCLUDE environment variable is
+        /// set, it is used as a list of files to include instead of the files from the current
+        /// directory.
+        /// </summary>
+        private static FileInfo[] GetDefaultFiles()
+        {
+            // Check if the LGTM_INDEX_INCLUDE environmen variable exists
+            var include = System.Environment.GetEnvironmentVariable("LGTM_INDEX_INCLUDE");
```

Comment:
This is great. Reminds me that there is LGTM_INDEX_EXCLUDES as well, but we can write that up once there is an ask for it

Reply:
Indeed, that's a logical next step 😄